notebooks/Get Species data from multiple sorces (spocc).qmd

```diff
@@ -0,0 +1,267 @@
+![](https://raw.githubusercontent.com/EcoCommons-Australia-2024-2026/ec-notebook_site/main/images/notebooks_banner_withframe.png)
+
+# Getting Shorebirds data in Australia Using Open Data and `spocc`
+
+Author details: Abhimanyu Raj Singh
+
+Editor details: Dr Sebastian Lopez Marcano
+
+Contact details: support\@ecocommons.org.au
+
+Copyright statement: This script is the product of the EcoCommons platform. Please refer to the EcoCommons website for more details: https://www.ecocommons.org.au/
+
+Date: February 2025
+
+# Script information
+
+This notebook was developed by the EcoCommons team to support researchers and conservationists in analyzing shorebird distribution data using open data sources. It demonstrates how to query, clean, visualize, and save species occurrence data using the `spocc` package in R.
```

Comment:
Also add the reference to the package give the central focus on this package across the notebook. 
```suggestion
This notebook was developed by the EcoCommons team to support researchers and conservationists in analysing shorebird distribution data using open data sources. It demonstrates how to query, clean, visualise, and save species occurrence data using the `spocc` package in R.
```

Reply:
Add in text reference